src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1263,10 +1337,209 @@ public static void DCJS_TypeWithKnownTypeAttributeAndListOfInterfaceMember()
     }
 
     [Fact]
+    public static void DCJS_Tuple()
+    {
+        DCJS_Tuple1();
+        DCJS_Tuple2();
+        DCJS_Tuple3();
+        DCJS_Tuple4();
+        DCJS_Tuple5();
+        DCJS_Tuple6();
+        DCJS_Tuple7();
+        DCJS_Tuple8();
+    }
```

Comment:
All these are Tuple related and there's no extract value separating them, only overheads. Combining them reduces the number of test cases and avoid related overheads.